### ISSUE TITLE
Airgraph-ng fails to process AP with special characters

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1037]
**Reported by misterx on 7 Oct 2012 19:55 UTC**

```
root @ bt:~# airgraph-ng -i dump-01.csv -g cpg -o derby.png

**** WARNING Images can be large, up to 12 Feet by 12 Feet****
Creating your Graph using, dump-01.csv and writing to, derby.png
Depending on your system this can take a bit. Please standby......
Error: airGconfig.dot:157: syntax error near line 157
context:  >>>   "''"<XSS>( <<< {})" [color=blue fontsize=9](label="''"<XSS>({})");
Warning: airGconfig.dot:158: string ran past end of line
```

**Comment by aircrack-ng** [comment]
**Attachment by misterx on 7 Oct 2012 19:56 UTC**

Uploaded file [dump-01.csv](https://github.com/aircrack-ng/trac-attachments/blob/master/1037/20121007-195648_000-dump-01.csv)

Use this file to reproduce the error.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 7 Oct 2012 19:55 UTC**

Fixed in #3045
